tests/ios/Podfile.lock

```diff
@@ -1,733 +0,0 @@
-PODS:
```

Comment:
removing this causes conflicts on rebase to master but it's just a `git add tests/ios/Podfile.lock` fix.

Reply:
I'm sorry to see you late. You can rest assured that I'll send you feedback after I review it.